### ISSUE TITLE
device alias gets erased upon zerotier restarts

### ISSUE THREAD

**Issue by CPSconcept** [key-networks/ztncui#27]
Hello,  
i encounter an issue where if i remove a device from a network, 
i can still see it, this is considered "normal behavior", as described in issue #16 , a simple fix to this is to restart zerotier-one like this:
`sudo systemctl restart zerotier-one` 
this will wipe the device from ram, as described here:
https://github.com/zerotier/ZeroTierOne/issues/859 .
However, by doing this, in all the other networks where the device is present, its alias gets erased.
I don't think i have to explain why this is annoying as it makes it really hard to identify devices across multiple networks.
thanks in advance for your response, have a nice day.

**Comment by key-networks** [comment]
@CPSconcept, as indicated in https://github.com/zerotier/ZeroTierOne/issues/859 this problem has been fixed in the dev branch of ZeroTierOne on 12 June 2019.  For some unknown reason, the fix has not been released yet in the 1.4 branch of ZeroTierOne.

The proper place for this problem to be fixed is in ZeroTierOne itself.  It could cause problems down the line if we attempted to build a work-around in ztncui at this stage, as you have seen where the alias gets erased in all other networks with the current manual work-around.
